tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -360,7 +360,13 @@ ipfwd/test_dir_bcast.py:
   skip:
     reason: "Unsupported topology."
     conditions:
-      - "topo_type not in ['t0', 'm0']"
+      - "topo_type not in ['t0', 'm0', 'dualtor']"
+
+ipfwd/test_dir_bcast.py:
+  xfail:
```

Comment:
The `xfail` mark need to be specified under the same `ipfwd/test_dir_bcast.py` entry. Otherwise, we will have duplicated entries in this yaml file.

Reply:
OK, I will change it